[BUG]: conf.py generated by sphinx-quickstart does not pass super-linter

### Description of the bug

The current rules enforced by the [Super-Linter](https://github.com/github/super-linter) workflow prevent well written Python scripts (such as those generated by `sphinx-quickstart`) to pass

### Steps To Reproduce

1. Create a test branch starting from `main`
   ```bash
   git checkout main
   git pull --all --prune
   git checkout -b test/issue-61
   ```
2. Install the [Sphinx](https://www.sphinx-doc.org/) tool and run the `sphinx-quickstart` script
   ```bash
   pip3 install sphinx
   PATH=$HOME/.local/bin:$PATH
   cd docs
   sphinx-quickstart
   ```
3. Do `git add` the `git commit` of the files just created by `sphinx-quickstart`
4. Wait for Super-Linter to perform the check on the test branch

### Expected results

No errors

### Actual results

Excerpt from `PYTHON_BLACK` errors

```text
2022-01-19 18:27:42 [INFO]   
2022-01-19 18:27:42 [INFO]   ----------------------------------------------
2022-01-19 18:27:42 [INFO]   ----------------------------------------------
2022-01-19 18:27:42 [INFO]   Linting [PYTHON_BLACK] files...
2022-01-19 18:27:42 [INFO]   ----------------------------------------------
2022-01-19 18:27:42 [INFO]   ----------------------------------------------
2022-01-19 18:27:42 [INFO]   ---------------------------
2022-01-19 18:27:42 [INFO]   File:[/github/workspace/docs/conf.py]
2022-01-19 18:27:42 [ERROR]   Found errors in [black] linter!
2022-01-19 18:27:42 [ERROR]   Error code: 1. Command output:
------
--- /github/workspace/docs/conf.py	2022-01-19 18:27:29.686837 +0000
+++ /github/workspace/docs/conf.py	2022-01-19 18:27:42.747600 +0000
@@ -15,41 +15,39 @@
 # sys.path.insert(0, os.path.abspath('.'))
 
 
 # -- Project information -----------------------------------------------------
 
-project = 'ARNEIS'
-copyright = '2022, The B-AROL-O Team'
-author = 'The B-AROL-O Team'
+project = "ARNEIS"
+copyright = "2022, The B-AROL-O Team"
+author = "The B-AROL-O Team"
 
 
 # -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = [
-    'myst_parser'
-]
+extensions = ["myst_parser"]
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = ["_templates"]
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
 
 # -- Options for HTML output -------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+html_theme = "alabaster"
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_static_path = ["_static"]
 
 # EOF
would reformat /github/workspace/docs/conf.py
Oh no! 💥 💔 💥
1 file would be reformatted.
------
2022-01-19 18:27:42 [INFO]   
```

### Additional Information

_No response_

--- a/.github/linters/.python-black
+++ b/.github/linters/.python-black
@@ -1,0 +1,4 @@
+[tool.black]
+skip-string-normalization
+
+# EOF

